Cargo.toml

```diff
@@ -16,15 +16,15 @@ sp1-helios-program = { path = "program" }
 sp1-helios-primitives = { path = "primitives" }
 
 # helios
-helios = { git = "https://github.com/a16z/helios", version = "0.7.0" }
-helios-consensus-core = { git = "https://github.com/a16z/helios", version = "0.7.0" }
-helios-ethereum = { git = "https://github.com/a16z/helios", version = "0.7.0" }
+helios = { git = "https://github.com/a16z/helios", rev = "59135e86e8d28b11aed217f636ebc25725e511ff" }
+helios-consensus-core = { git = "https://github.com/a16z/helios", rev = "59135e86e8d28b11aed217f636ebc25725e511ff" }
+helios-ethereum = { git = "https://github.com/a16z/helios", rev = "59135e86e8d28b11aed217f636ebc25725e511ff" }
```

Comment:
Do you need to use this for the `serde` fix? Can you add a comment how this will be released in `0.8.1`?